corehq/tabs/uitab.py

```diff
@@ -8,6 +8,18 @@
 from django.conf import settings
 
 
+def url_is_location_safe(url):
+    from corehq.apps.users.location_access_restrictions import is_location_safe
+    try:
+        match = resolve(url)
+    except Resolver404:
+        return False
+    else:
+        if is_location_safe(match.func, match.args, match.kwargs):
+            return True
+        return False
```

Comment:
The above four lines could be reduced to
```py
return is_location_safe(match.func, match.args, match.kwargs)
```
(and put at same level of indentation as `else`)